damus/Views/SetupView.swift

```diff
@@ -34,17 +34,12 @@ struct SetupView: View {
                         .shadow(color: DamusColors.purple, radius: 2)
                         .frame(width: 56, height: 56, alignment: .center)
                         .padding(.top, 20.0)
-                    
-                    HStack {
-                        Text("Welcome to", comment: "Welcome text shown on the first screen when user is not logged in.")
```

Comment:
The issue is that we can't split a sentence up into two like this. It makes it impossible for translators to translate, especially in languages that aren't grammatically the same as English. The compiler also doesn't like me embedding a gradient Text component inside a non-gradient Text component, for whatever reason.

I checked with Rob on the design, and he voted for applying the gradient on the entire string.